src/main/java/org/broadinstitute/hellbender/utils/genotyper/ReadLikelihoods.java

```diff
@@ -1339,13 +1300,11 @@ private void removeSampleReads(final int sampleIndex, final Collection<GATKRead>
     public PerReadAlleleLikelihoodMap toPerReadAlleleLikelihoodMap(final int sampleIndex) {
         Utils.validIndex(sampleIndex, samples.numberOfSamples());
         final PerReadAlleleLikelihoodMap result = new PerReadAlleleLikelihoodMap();
-        final int alleleCount = alleles.numberOfAlleles();
         final GATKRead[] sampleReads = readsBySampleIndex[sampleIndex];
-        final int sampleReadCount = sampleReads.length;
-        for (int a = 0; a < alleleCount; a++) {
+        for (int a = 0; a < alleles.numberOfAlleles(); a++) {
             final A allele = alleles.getAllele(a);
             final double[] readLikelihoods = valuesBySampleIndex[sampleIndex][a];
-            for (int r = 0; r < sampleReadCount; r++)
+            for (int r = 0; r < sampleReads.length; r++)
```

Comment:
just making sure you are aware of performance impact of this - accessing `.length` in a tight loop is not free